api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.api.generation.common
+
+object IgnoredParameters {
+  val deprecated: Seq[String] = Seq(
+    "r2_stopping", // All
+    "max_confusion_matrix_size", // Deep Learning
+    "col_major") // Deep Learning
+
+  val implementedInParent: Seq[String] = Seq(
+    "model_id",
+    "training_frame",
+    "validation_frame",
+    "response_column",
+    "fold_column",
+    "weights_column",
+    "offset_column",
+    "seed",
+    "nfolds",
+    "distribution")
+
+  val unimplemented = Seq(
```

Comment:
This list of parameters must be  implement manually. Will do in another PRs. IMHO, this PR is quite big.